docs/settings/security-settings.asciidoc

```diff
@@ -239,3 +239,107 @@ The format is a string of `<count>[ms\|s\|m\|h\|d\|w\|M\|Y]` (e.g. '20m', '24h',
 ============
 
 |===
+
+[float]
+[[audit-logging-settings]]
+===== Audit logging settings
+
+You can enable audit logging to support compliance, accountability and security. When enabled Kibana will capture:
+
+- Who performed an action
+- What action was performed
+- When it occurred
+
+For more details and a reference of audit events see <<xpack-security-audit-logging>>.
+
+[cols="2*<"]
+|===
+| `xpack.security.audit.enabled`
+| Set to `true` to enable audit logging for security events. *Default:* `false`
+
+| `xpack.security.audit.ignore_filters[]` (beta)
+| List of filters that determine which events should be excluded from the audit log. An event will get filtered out if at least one of the provided filters matches.
+|===
+
+For example:
+
+[source,yaml]
+----------------------------------------
+xpack.security.audit.ignore_filters:
+- actions: [http_request] <1>
+- categories: [database]
+  types: [creation, change, deletion] <2>
+----------------------------------------
+<1> Filters out HTTP request events
+<2> Filters out any data write events
+
+[cols="2*<"]
+|===
+| `xpack.security.audit.ignore_filters[].actions[]` (beta)
+| List of values matched against the `event.action` field of an audit event. See <<xpack-security-audit-logging>> for a list of available events.
+
+| `xpack.security.audit.ignore_filters[].categories[]` (beta)
+| List of values matched against the `event.category` field of an audit event. See https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-category.html[ECS categorization field] for allowed values.
+
+| `xpack.security.audit.ignore_filters[].types[]` (beta)
+| List of values matched against the `event.type` field of an audit event. See https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-type.html[ECS type field] for allowed values.
+
+| `xpack.security.audit.ignore_filters[].outcomes[]` (beta)
+| List of values matched against the `event.outcome` field of an audit event. See https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-outcome.html[ECS outcome field] for allowed values.
+
+| `xpack.security.audit.appender` (beta)
+| Optional. Specifies where audit logs should be written to and how they should be formatted. *Default:* console
+
+2+a|
+[NOTE]
+============
+If you set `xpack.security.audit.appender` the legacy audit logger will be disabled and the new audit logger will be used instead.
+============
+|===
+
+For example:
+
+[source,yaml]
+----------------------------------------
+xpack.security.audit.appender:
+  kind: file
+  path: /path/to/audit.log
+  layout:
+    kind: json
+----------------------------------------
+
+[cols="2*<"]
+|===
+| `xpack.security.audit.appender.kind` (beta)
+| Mandatory. Specifies where audit logs should be written to. Allowed values are `console` or `file` appender.
+|===
+
+[float]
+[[file-appender-settings]]
```

Comment:
Since this anchor is global to all of the kibana docs, we should probably make it more specific so that we don't collide with other anchors in the future

```suggestion
[[audit-logging-file-appender-settings]]
```